whirlpool/oven.py

```diff
@@ -362,11 +363,14 @@ async def stop_cook(self, cavity: Cavity = Cavity.Upper):
                 CAVITY_PREFIX_MAP[cavity]
                 + "_"
                 + ATTR_POSTFIX_SET_OPERATION: COOK_OPERATION_MAP[CookOperation.Cancel]
-            }
+            },
```

Comment:
```suggestion
            }
```

Reply:
Fixed in 046dd236a6a5aec3a2a8bad9062df3d92bc41a0b